joomla/joomla-cms
# [a11y] Table header for articles category list

Pull Request for Issue #.

### Summary of Changes
To be accessible a table must have a header. 
See for examle this checklist: https://dequeuniversity.com/checklists/web/tables

Joomla provides a param show_headings for frontend tables. This removes the table header. 

This PR sets the table header visually-hidden so it is still present for assistive technologies. 

I open some similar PRs to make testing easier. 


### Testing Instructions
You need at least one article 
Make a menu Item of type category list.
Set param table_headings in tab "list layouts" to "hide"
See the view and sourcecode ot the table in frontend 

### Actual result BEFORE applying this Pull Request
There is no table heading visible, also in source code.


### Expected result AFTER applying this Pull Request
There is no table heading visible, but it is in sourcecode with class="visually-hidden"

### Link to documentations
Please select:
